packages/fxa-payments-server/src/components/PayPalButton/index.tsx

```diff
@@ -112,7 +112,7 @@ export const PaypalButton = ({
           ...apiClient,
           ...apiClientOverrides,
         };
-        if (!customer) {
+        if (!customer?.payment_provider) {
```

Comment:
The front-end gets customers back with no payment_provider? I'm curious what the flag here is meant to represent, and whether a helper method on the customer could make that more obvious when reading.